src/scss/_themes.scss

```diff
@@ -19,39 +19,80 @@
 	}
 }
 
-@mixin oTeaserCollectionSpecial() {
+@mixin oTeaserCollectionSpecial($colour) {
 	position: relative;
 	border-top: 0;
 	margin-bottom: 50px;
 
 	&:after {
 		content: '';
 		position: absolute;
-		top: 0px;
+		top: 0;
 		left: 50%;
 		z-index: -1;
 		width: 100vw;
 		height: 100%;
-		background: $_o-teaser-collection-color-special-bg;
+		background: $colour;
 		transform: translateX(-50%);
 	}
 }
 
+@mixin oTeaserCollectionAssassination {
+	@include oTeaserCollectionBigStory();
+
+	.o-teaser-collection--assassination-related {
+		@include oTeaserCollectionSpecial($_o-teaser-collection-color-assassination-bottom-bg);
+		z-index: 1;
+		margin-bottom: 0;
+		padding-left: 10px;
+
+		&:after {
+			left: 52%;
+		}
+
+		// modifies borders when stacked(S) and horizontal(M+)
+		.o-teaser-collection__item {
+			margin: 0;
+			border-color: rgba(255, 255, 255, 0.2);
```

Comment:
It would be easier to read these three lines as `border-bottom: 1px solid rgba(#fff, 0.2)` (replacing #fff with the oColors function) if possible.